monai/data/box_utils.py

```diff
@@ -572,3 +572,261 @@ def convert_box_to_standard_mode(
             box_convert_mode(boxes=boxes, src_mode="xxyyzz", dst_mode="xyzxyz")
     """
     return convert_box_mode(boxes=boxes, src_mode=mode, dst_mode=StandardMode())
+
+
+def check_boxes(boxes: NdarrayOrTensor) -> bool:
```

Comment:
I would suggest to change to a more concrete function name, like "check_box_values" or "is_valid_box_values".
What do you think?

Thanks.